integration_test.go

```diff
@@ -321,15 +320,13 @@ func newMockTransport() *MockTransport {
 	}
 }
 
-func mockHarvester(mt *MockTransport, common map[string]interface{}) (*instrumentation.MetricAggregator, *telemetry.Harvester) {
-	agg := instrumentation.NewMetricAggregator()
-	return agg, telemetry.NewHarvester(
+func mockHarvester(mt *MockTransport, common map[string]interface{}) *telemetry.Harvester {
```

Comment:
Need to return an error here and check it in test setup.

Reply:
👍